src/System.Linq.Expressions/tests/Variables/ParameterExpressionTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Jon Hanna. All rights reserved.
```

Comment:
@eerhardt, this is related: https://github.com/dotnet/coreclr/pull/1329#discussion_r36218185.